### DISCUSSION TITLE
zh/guide/drivers/mediatrack

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#93] [Announcements]
# zh/guide/drivers/mediatrack

访问令牌 登录后可以在请求头中获取 token 项目编号 从官网网址获取： Project id 根文件夹 ID 登录后从请求中获取 id

https://alist.nn.ci/zh/guide/drivers/mediatrack.html

**Comment by qpzmwasdjkl** [comment]
分秒帧提示：Failed init storage: 登录已失效，请重新登录，多次登录都是这样

**Reply by cditmoo** [reply]
我也是

**Reply by HC-CPU** [reply]
把根文件夹中开头的Bearer和后面的空格删去就行了

**Comment by owenxuan** [comment]
failed link: failed get link: 地区网络故障，请联系客服 
打开视频或照片就显示这个

**Reply by CXFKanesx** [reply]
#我也是，有无大佬解决一下啊😭

**Comment by endercatone** [comment]
failed link: failed get link: 地区网络故障，请联系客服
